### PR TITLE
confusion between dropout rate and keep_prob rate

### PR DESCRIPTION
**Pull request by chg0901**
Possible confusion between dropout rate vs keep_prob rate #122

### CODE REVIEW

**Comment by kkweon** [comment]
This file uses `tf.nn.dropout(...)`
```python
tf.nn.dropout(L1, keep_prob=keep_prob)
```
uses `keep_prob` so it's correct to use 0.7.

#122 issue was about `tf.layers.dropout(...)` which uses the dropout rate not `keep_prob` as in `tf.nn.dropout`
